proto/cacheclient.proto

```diff
@@ -177,6 +180,29 @@ message _SetIfNotExistsResponse {
   message _NotStored { }
 }
 
+// tokens are restricted to 8 bytes
```

Comment:
Can the Javascript SDK handle uint64 or is it better to model the token as byte arrays and restrict them to 8 bytes in the implementation?